.github/workflows/release.yml

```diff
@@ -0,0 +1,160 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: Tag for release
+        required: true
+
+jobs:
+  build-binary:
+    name: Build Binary
+    runs-on: self-hosted
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.inputs.tag }}
+      - name: Setup Rust
+        run: rustup show
+      - name: Build
+        run: |
+          cargo build --locked --release --features turing-node
+          mkdir -p artifacts/
+          cp target/release/oak-collator artifacts/
+      - name: Build debian
+        run: |
+          cargo install cargo-deb
+          cargo deb -p oak-node
+          cp target/debian/*.deb artifacts/
+      - name: Upload binary
+        uses: actions/upload-artifact@v2
+        with:
+          name: oak-collator
+          path: artifacts/
+  build-runtime:
+    name: Build Runtime
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        chain: ["turing"]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.inputs.tag }}
+      - name: Build
+        id: srtool_build
+        uses: chevdor/srtool-actions@v0.4.0
+        with:
+          chain: ${{ matrix.chain }}
+      - name: Summary
+        run: |
+          echo '${{ steps.srtool_build.outputs.json }}' | jq . > ${{ matrix.chain }}-srtool-digest.json
+          cat ${{ matrix.chain }}-srtool-digest.json
+          cp ${{ steps.srtool_build.outputs.wasm_compressed }} ${{ matrix.chain }}-runtime.wasm
+      - name: Upload runtime
+        uses: actions/upload-artifact@v2
+        with:
+          name: turing-runtime
+          path: |
+            ${{ matrix.chain }}-runtime.wasm
+            ${{ matrix.chain }}-srtool-digest.json
+  build-docker:
+    name: Build Docker
+    runs-on: ubuntu-latest
+    needs: build-binary
+    steps:
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.inputs.tag }}
+      - name: Download binary
+        uses: actions/download-artifact@v2
+        with:
+          name: oak-collator
```

Comment:
Isn't this a zip with both the binary and the deb file? Doesn't it need to be unpackaged before doing the docker build?
I didn't see anything in the download-artifact action documentation about this implicitly unpacking but maybe it does.

Reply:
The download-artifact action is doing this for us.